UC2/pom.xml

```diff
@@ -9,12 +9,12 @@
 	</parent>
 	<artifactId>UC2</artifactId>
 	<name>UC2</name>
-	<version>0.2.0</version>
```

Comment:
Version 0.3.0 of UC2 does not exists; future changes are not as relevant and they only change the patch.